spec/services/metadata_cloud_service_spec.rb

```diff
@@ -2,28 +2,31 @@
 require "rails_helper"
```

Comment:
Unless there's more substantive changes to this file, can you re-pull and just add the "around do"? The changes I made were to make it so that this passes both on and off the VPN.

Reply:
Fixed